test/plugins/features/movies/controller.test.js

```diff
@@ -23,5 +24,55 @@ describe('movie controller', () => {
 
   });
 
+  describe('list', () => {
+
+    const movie_list = [
+      {
+        title: 'Argo',
+        release_year: 2011
+      },
+      {
+        title: 'Argo2',
+        release_year: 2012
+      },
+      {
+        title: 'Bedazzled',
+        release_year: 2007
+      }
+    ];
+
+    beforeEach(() => {
+      return Knex.raw('TRUNCATE movies CASCADE;')
+        .then(() => {
+          return Knex('movies').insert(movie_list);
+        });
+    });
+
+    it('List all movies. No filters specified', () => {
```

Comment:
Nit: Usually we name tests using TDD-inspired syntax so that when the full test name is constructed, based on the nested `it`'s and `describe`'s, the name reads like one sentence.

In this case if the test were renamed `lists all movies with no filters specified`, the full test name would be `list lists all moves with no filters specified`. See [api](https://github.com/lob/lob-api/blob/master/test/plugins/features/areas/index.test.js#L73) for further examples.